packages/flutter/lib/src/material/text_field.dart

```diff
@@ -793,6 +795,12 @@ class TextField extends StatefulWidget {
   /// configuration, then [materialMisspelledTextStyle] is used by default.
   final SpellCheckConfiguration? spellCheckConfiguration;
 
+  /// {@macro flutter.rendering.RenderEditablePainter}
```

Comment:
The 2 painters may need to be documented differently since it looks like they have different paint orders.

Reply:
Also the linked documentation seems to focus on what the interface is for in general, instead of what each painter does.

Reply:
Make sense, documented.